idm/meta/dao/sql/ns-sql.go

```diff
@@ -28,14 +28,14 @@ import (
 	"gorm.io/gorm/clause"
 	"gorm.io/gorm/schema"
 
-	structpb "github.com/golang/protobuf/ptypes/struct"
 	"github.com/pydio/cells/v5/common/errors"
 	"github.com/pydio/cells/v5/common/proto/idm"
 	"github.com/pydio/cells/v5/common/proto/service"
 	"github.com/pydio/cells/v5/common/storage/sql"
 	"github.com/pydio/cells/v5/common/storage/sql/resources"
 	"github.com/pydio/cells/v5/common/telemetry/log"
 	"github.com/pydio/cells/v5/idm/meta"
+	"google.golang.org/protobuf/types/known/structpb"
```

Comment:
imports ordering do you run goimports-reviser ? 
Should group imports by stdlib, external imports, "internal" imports (a.k.a cells)